site/assets/scss/docs.scss

```diff
@@ -51,3 +51,8 @@
 @import "syntax";
 @import "anchor";
 @import "algolia";
+
+
+.offcanvas{
+  z-index: 1072 // over .bd-subnavbar
+}
```

Comment:
Move these styles to the appropriate section in `_component-examples.scss`, please. Also remember to add a space before the opening curly brace.